lib/Adapter/Php/InMemory/InMemoryIndex.php

```diff
@@ -54,4 +91,9 @@ public function exists(): bool
     {
         return $this->repository->lastUpdate !== 0;
```

Comment:
This timestamp is used to detect changes _since_ the indexer was last executed. So this would _always_ (!) be more than 1 second. After the initial index we switch to watching. But if you close your editor and start up again, it will only index files modified after the last index update.